http/src/main/java/com/hazelcast/jet/contrib/http/impl/HttpListenerSinkContext.java

```diff
@@ -142,8 +146,13 @@ private void putToBuffer(T item) {
         private final WebSocketProtocolHandshakeHandler handler;
         private final Set<WebSocketChannel> peerConnections;
 
-        WebSocketSinkHttpHandler() {
+        WebSocketSinkHttpHandler(ConcurrentLinkedQueue<String> messageBuffer) {
             handler = Handlers.websocket((exchange, channel) -> {
+                String message = messageBuffer.poll();
+                while (message != null) {
+                    send(message);
+                    message = messageBuffer.poll();
+                }
```

Comment:
Can't we just call `drainBuffer`? Also, isn't NPE possible here if the `messageBuffer` is null?

Reply:
drain is not available in `ConcurrentLinkedQueue`, handled NPE with a noop message buffer